packages/manager/src/features/Profile/Settings/PreferenceEditor.tsx

```diff
@@ -24,23 +20,24 @@ const PreferenceEditor: React.FC<CombinedProps> = (props) => {
 
   React.useEffect(() => {
     setLoading(true);
-    getUserPreferences()
+    refetchPreferences()
+      .then(({ data: userPreferences }) => userPreferences ?? Promise.reject())
       .then((userPreferences) => {
         setLoading(false);
         setUserPrefs(JSON.stringify(userPreferences, null, 2));
       })
       .catch(() => {
         setErrorMessage('Unable to load user preferences');
       });
-  }, [getUserPreferences]);
+  }, []);
 
   const handleSavePreferences = () => {
     try {
       const parsed = JSON.parse(userPrefs);
       setSubmitting(true);
       setSuccessMessage('');
       setErrorMessage('');
-      updateUserPreferences(parsed as any)
+      updatePreferences(parsed as any)
```

Comment:
Type is already `any` so can just remove the cast

```suggestion
      updatePreferences(parsed)
```